### ISSUE TITLE
too many warning and logs!

### ISSUE THREAD

**Issue by FullDataAlchemist** [pyroscope-io/pyroscope#1382]
Hi. I recently upgraded my pyroscope. I have no idea, but nothing works! 
I am using pyroscope docker image 0.25.1  (i was using 0.19 before)
pyroscope client python 0.7.1 (i was using 0.6 before)
I am using just this line for initializing the pyroscope : 
`pyroscope.configure(application_name, server_address)`
I repeatedly get these warnings.
```
 2022-08-10T12:27:10.000Z WARN  Pyroscope::Timer   > Failed to send event to listener @ Sender { .. } - sending on a closed channel
 2022-08-10T12:27:10.003Z INFO  Pyroscope::Session > Creating Session
 2022-08-10T12:27:10.003Z INFO  Pyroscope::Session > Sending Session: 1660134420 - 1660134430
````
I set the `sample_rate` to 10, and this message repeats every 10-milliseconds.

**Comment by omarabid** [comment]
@FullDataAlchemist That's not exactly an error (it's a Warning) and a harmless one. Can you confirm whether you are getting data on the Pyroscope Server or not?

**Comment by siumingdev** [comment]
I am also getting a lot of these warnings, my pyroscope python client version is 0.7.2 and server docker image tag is latest (spinned up yesterday, so i guess it is 0.25.1)
Actually what does it mean by these warning messages? Does it harm the performance? How can we get rid of them?

**Comment by Rperry2174** [comment]
Thanks for reporting @siumingdev we're working on fixing now!

**Comment by petethepig** [comment]
Fixed it in [0.8.0](https://pypi.org/project/pyroscope-io/0.8.0/)

@FullDataAlchemist @siumingdev let us know if it's working for you!
